Blackjack.py

```diff
@@ -0,0 +1,282 @@
+            self.bet = 0
+
+        def win_bet(self):
+            self.total += self.bet
+
+        def lose_bet(self):
+            self.total -= self.bet
+
+
+    def take_bet(chips):
+
+        while True:
+            try:
+                chips.bet = int(input("Place your bet Player: "))
+
+            except ValueError:
+                print("You must bet an integer value")
+
+            else:
+                if chips.bet > chips.total:
+                    print("You exceeded your total of {chips.total}")
+                else:
+                    break
+
+
+    def hit(deck, hand):
+        hand.add_card(deck.deal())
+        hand.adjust_ace()
+
+
+    def hit_or_stand(deck, hand):
+        global playing
+
+        while True:
+
+            turn = input("\nWill you (hit) or (stand)? (please type one of the following under paranthesis)\n")
+
+            if turn == "hit":
+                hit(deck, hand)
+            elif turn == "stand":
+                playing = False
+            else:
+                print('input must be either "hit" or "stand"! Try again')
+                continue
+            break
+
+
+    def show_some(player, dealer):
+        print("\nShow me only some cards yo ;)")
+        print(f"\nDealer's hand:\nhidden\n{dealer.all_cards[1]}")
+        print("\nPlayer's hand:", *player.all_cards, player, sep="\n")
+
+
+    def show_all(player, dealer):
+        print("Lay it all on the floor , please yo!")
+        print("\nDealer's Hand: \n", *dealer.all_cards, dealer, sep="\n")
+        print("\nPlayer's Hand: \n", *player.all_cards, player, sep="\n")
+
+
+    def player_busts(player, player_chips):
+        print("*****************************")
+        print("Player busts")
+        print("*****************************")
+        player_chips.lose_bet()
+        # player.all_cards = []
+
+
+    def player_wins(player, player_chips):
+        print("*****************************")
+        print("Player wins!")
+        print("*****************************")
+        player_chips.win_bet()
+        # player.all_cards = []
+
+
+    def dealer_busts(dealer, dealer_chips):
+        print("*****************************")
+        print("Dealer busts")
+        print("*****************************")
+        dealer_chips.lose_bet()
+        # dealer.all_cards = []
+
+
+    def dealer_wins(dealer, dealer_chips):
+        print("*****************************")
+        print("Dealer Wins!")
+        print("*****************************")
+        dealer_chips.win_bet()
+        # dealer.all_cards = []
+
+
+    def push():
+        print("****************************************************")
+        print("Its a tie, both Player and Dealer Push Bay-Bee")
+        print("****************************************************")
+        pass
+
+
+    the_deck = Deck()
+    # player = Hand()
+    # dealer = Hand()
+
+    player_chips = Chips()
+    dealer_chips = Chips()

```

Comment:
You could still have `Player` class to hold all the relevant info, but they get a `Hand` and a `Chips` attribute inside them.

```
class player():

    def __init__():
        self.hand = Hand()
        self.chips = Chips()

# then you could still access everything with:
player.hand
dealer.hand

player.chips
dealer.chips

```